src/rdkafka_cgrp.c

```diff
@@ -490,13 +490,13 @@ static void rd_kafka_cgrp_handle_FindCoordinator (rd_kafka_t *rk,
 
         if (ErrorCode == RD_KAFKA_RESP_ERR_GROUP_COORDINATOR_NOT_AVAILABLE)
                 rd_kafka_cgrp_coord_update(rkcg, -1);
-	else {
+        else {
                 if (rkcg->rkcg_last_err != ErrorCode) {
-                        rd_kafka_q_op_err(rkcg->rkcg_q,
-                                          RD_KAFKA_OP_CONSUMER_ERR,
-                                          ErrorCode, 0, NULL, 0,
-                                          "FindCoordinator response error: %s",
-                                          errstr);
+                        rd_kafka_consumer_err(
```

Comment:
can you give a precise scenario where the user would be worse off if this was sent to the error handler instead?

i need an example, i'm failing to get the benefit.

Reply:
I can't! Let's revise all error propagation at a later time.